README.md

```diff
@@ -68,7 +71,8 @@ Currently supported:
 - [x] Print output in `Prettytable` format
 - [ ] Allow search for Entertainment Channels
 - [x] Provide option to save details as PDF
-- [ ] Send notification to the user
+- [ ] Create an account of the user
```

Comment:
If we would have a user account then through email or other platform it would be easy to send the notification to the user.